lib/easypost/carrier_account.rb

```diff
@@ -6,4 +6,19 @@ class EasyPost::CarrierAccount < EasyPost::Resource
   def self.types
     EasyPost::CarrierType.all
   end
+
+  def self.create(params = {}, api_key = nil)
+    wrapped_params = {}
+    wrapped_params[class_name.to_sym] = params
+
+    # For Ups and Fedex the endpoint is different
+    create_url = if params[:type] == 'UpsAccount' || params[:type] == 'FedexAccount'
+                   "#{url}/register"
```

Comment:
A couple points of feedback here that we need to get resolved before we can move on this:

1. Please make this a constant list of strings containing `FedexAccount` and `UpsAccount`, then check that the `:type` is included in this constant. This will make it easier for us to extend it in the future.

Reply:
Done